pkgs/top-level/all-packages.nix

```diff
@@ -15880,6 +15880,7 @@ with pkgs;
   openjdk_headless = jdk_headless;
 
   graalvmCEPackages = callPackage ../development/compilers/graalvm/community-edition { };
+  graalvmPackages = recurseIntoAttrs (callPackage ../development/compilers/graalvm { });
```

Comment:
I don't like that we have 2 different `graalvmPackages`.

I think since we are introducing Oracle GraamVM again, it is a good time to move everything to `graalvmPackages` and make `graalvmCEPackages` an alias to `graalvmPackages`, and deprecate and eventually remove `graalvmCEPackages`.

Reply:
I can do that, but can you be more specific? Do we want `graalvmPackages` to temporarily be the CE packages and also `graalvmPackages.oracle` + `graalvmPackages.ce` as the long-term, stable names?

Do we also want this nesting for JDK packages or is `pkgs.oracle-graalvm` fine as it is in the current version of the PR?

Reply:
> I can do that, but can you be more specific? Do we want graalvmPackages to temporarily be the CE packages and also graalvmPackages.oracle + graalvmPackages.ce as the long-term, stable names?

Yes. But probably the names will be `graalvmPackages.graalvm-oracle` and `graalvmPackages.graalvm-ce`, because there are other packages inside `graalvmCEPackages` too.

Reply:
I've updated the PR, please let me know what you think